README.md

```diff
@@ -77,6 +77,10 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | `bbb_monitoring_all_in_one_htpasswd_user` | The user for the htpasswd - _(required)_ if external | `Undefined` | |
 | `bbb_monitoring_all_in_one_htpasswd` | The password for the htpasswd - _(required)_ if external | `Undefined` | |
 | `bbb_monitoring_recordings_from_disk` | Collect recordings metrics by querying the disk instead of the API. See [this](https://bigbluebutton-exporter.greenstatic.dev/exporter-user-guide/#optimizations) for details. | `true` |
+| `bbb_monitoring_systemd_enable` | Deploy monitoring as systemd service(not recommended) | `false` | Works only when `bbb_monitoring_all_in_one_enable` is `false` |
+| `bbb_monitoring_systemd_version` | Version of the (greenstatic/bigbluebutton-exporter)[https://github.com/greenstatic/bigbluebutton-exporter] | `HEAD` | Could be a branch or tag |
+| `bbb_monitoring_systemd_directory` | Installation directory for git repository | `"/opt/bigbluebutton-exporter"` | |
+| `bbb_monitoring_systemd_external` | Enable exposure to nginx | `false` | |
```

Comment:
Please add `bbb_monitoring_systemd_port` and `bbb_monitoring_systemd_bind_ip` variables. They are not documented in the systemd exporter... but can be found in the code (`PORT` and `BIND_IP` environment variable):

https://github.com/greenstatic/bigbluebutton-exporter/blob/6a44d9a89028d95019231cfdaa4480b2b616e862/bbb-exporter/settings.py#L37-L38

Reply:
@e1mo I added the variables `bbb_monitoring_systemd_port` and `bbb_monitoring_systemd_bind_ip`
Can you please check the changes again?

Reply:
@e1mo I added the variables bbb_monitoring_systemd_port and bbb_monitoring_systemd_bind_ip